app/client/templates/SignupRules.tsx

```diff
@@ -0,0 +1,53 @@
+import { Layout } from "@client/components/Layout";
+import React from "react";
+
+export const Template = () => {
```

Comment:
If you will decide to use `react-markdown` above, you can do it here too

Reply:
Done